src/optimization/solution_assessment.jl

```diff
@@ -632,8 +632,8 @@ function simulated_annealing(
         obj_current = obj_best
         static_ctr = 0
 
-        @info "\n\tCluster\t\t$(cluster_set[clust_idx].id)\n\t" *
-              "Iteration \tBest Value \t\tTemp\n\t0\t\t$obj_best\t$temp"
+        @info """\n\tCluster\t\t$(cluster_set[clust_idx].id)\n\tIteration \tBest Value \
+        \t\tTemp\n\t0\t\t$obj_best\t$temp"""
```

Comment:
I don't understand why the overuse of escape sequences here, can you clarify? 

Why not just use it as a formatted string like normal?
Is it not rendering properly in the REPL?

```julia
@info """

Cluster: $(cluster_set[clust_idx].id)
Iteration \t Best Value \t t0
"""
```